insights/parsers/lsscsi.py

```diff
@@ -0,0 +1,95 @@
+"""
+LsSCSI - command ``/usr/bin/lsscsi``
+====================================
+
+This parser reads the output of ``/usr/bin/lsscsi`` into a list of dictionarys.
+Each item is a dictionary with six keys:
+
+* ``HCTL`` - the scsi_host,channel,target_number,LUN tuple
+* ``Peripheral-Type`` - the SCSI peripheral type
+* ``Vendor`` - the vendor name
+* ``Model`` - the model name
+* ``Revision`` - the revision string
+* ``Primary-Device-Node`` - the primary device node name
+
+Attributes:
```

Comment:
Please move the `Attributes` into the docsting of the class, to make the documentation looks good.